db/models/bitcoin.go

```diff
@@ -0,0 +1,114 @@
+package models
+
+import (
+	"log"
+
+	"../../db"
+)
+
+// Bitcoin -> Structure of the database
+type Bitcoin struct {
+	ID       int
+	Base     string
+	Currency string
+	Amount   string
+}
+
+// InsertBitcoinValue -> This inserts a new entry to the db
+func InsertBitcoinValue(base string, currency string, amount string) (Bitcoin, bool) {
+	database := db.GetConnection()
```

Comment:
Maybe we could define a global connection to avoid getting it with each request.